webapp/apps/taxbrain/helpers.py

```diff
@@ -69,7 +68,12 @@
     [         1,   '%', 1],  # "%age Tax Decrease",
     [         1,   '%', 1],  # "Share of Overall Change"
 ]
-TAXCALC_RESULTS_BIN_ROW_KEYS = dropq.dropq.bin_row_names
+TAXCALC_RESULTS_BIN_ROW_KEYS = [
+    "less_than_10", "ten_twenty", "twenty_thirty", "thirty_forty",
+    "forty_fifty", "fifty_seventyfive", "seventyfive_hundred",
+    "hundred_twohundred", "twohundred_fivehundred",
+    "fivehundred_thousand", "thousand_up", "all"
+]
```

Comment:
Yep, I did that to allow the application to start without dropq (which is not listed in the public requirements files). I moved the dropq-related changes to a new PR #51, which this PR is based off now.